packages/grafana-ui/src/components/Button/ToolbarButton.tsx

```diff
@@ -22,10 +22,12 @@ export interface Props extends HTMLAttributes<HTMLButtonElement> {
   narrow?: boolean;
   /** variant */
   variant?: ButtonVariant;
+  /** Changes border color to orange */
+  active?: boolean;
 }
 
 export const ToolbarButton = forwardRef<HTMLButtonElement, Props>(
-  ({ tooltip, icon, className, children, imgSrc, fullWidth, isOpen, narrow, variant, ...rest }, ref) => {
+  ({ tooltip, icon, className, children, imgSrc, fullWidth, isOpen, narrow, variant, active, ...rest }, ref) => {
```

Comment:
Should we adopt a convention where we prefix boolean props with `is` or `has`? Otherwise it looks a bit strange that we have `isOpen` and `active` props which are both booleans but have different spelling.

Reply:
yea, conflicted, I used to be all for `is` prefix for all booleans, but seems that as long as it's not a verb, many don't prefix it, and the default html attributes like disabled are not prefix like isDisabled, so there is some inconsistency there already.

Reply:
but a good topic for frontend weekly, would be good to discuss what everyone thinks and if we should have any guidelines for it